lib/api/debug.go

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2014 The Syncthing Authors.
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+package api
+
+import (
+	"os"
+	"strings"
+
+	"github.com/syncthing/syncthing/lib/logger"
+)
+
+var (
+	l = logger.DefaultLogger.NewFacility("http", "REST API")
+)
+
+func shouldDebugHTTP() bool {
+	return l.ShouldDebug("http")
+}
+
+func init() {
+	l.SetDebug("http", strings.Contains(os.Getenv("STTRACE"), "http") || os.Getenv("STTRACE") == "all")
```

Comment:
Rename to API? 🙃 🤔 
No? Too brave?

Reply:
Dis hidden dunno why

Reply:
Why not, we can still enable the new "api" facility when encountering "http" to stay backwards compatible. @calmh?

Reply:
No stated compatibility on debug facilities, rename is fine by me.